packages/components/textarea/src/textarea.ts

```diff
@@ -25,27 +22,35 @@ let nextUniqueId = 0;
  *
  * @slot textarea - The slot for the textarea element
  */
-export class Textarea extends FormControlMixin(HintMixin(LitElement)) {
+export class Textarea extends FormControlMixin(ScopedElementsMixin(LitElement)) {
   /** @private */
-  static override styles: CSSResultGroup = [validationStyles, hintStyles, styles];
+  static get scopedElements(): ScopedElementsMap {
+    return {
+      'sl-icon': Icon
```

Comment:
Icon should be also added as dependency in the textarea's package.json file?

Reply:
Good catch! I've fixed this for text-field as well.